packages/main/src/plugin/appearance-init.ts

```diff
@@ -65,6 +77,13 @@ export class AppearanceInit {
       if (e.key === APPEARANCE_FULL_KEY && typeof e.value === 'string') {
         this.updateNativeTheme(e.value);
       }
+
+      if (
+        e.key === `titleBar.${AppearanceSettings.SearchBar}` &&
```

Comment:
suggestion: `titleBar.${AppearanceSettings.SearchBar}` could be refactored with the occurence on line 64